subsys/bluetooth/host/iso.c

```diff
@@ -2073,6 +2130,19 @@ int bt_iso_big_terminate(struct bt_iso_big *big)
 }
 
 #if defined(CONFIG_BT_ISO_SYNC_RECEIVER)
+static void store_bis_sync_receiver_info(const struct bt_hci_evt_le_big_sync_established *evt,
+					 struct bt_iso_sync_receiver_info *info)
+{
+	info->latency = sys_get_le24(evt->latency);
+	info->max_subevent = evt->nse;
+	info->bn = evt->bn;
+	info->irc = evt->irc;
+	info->interval = sys_le16_to_cpu(evt->iso_interval);
+	/* Transform to ms */
+	info->pto = info->interval * evt->pto;
```

Comment:
Same as above - need to multiply by 1.25.

Reply:
See https://github.com/zephyrproject-rtos/zephyr/pull/45077#discussion_r857551073